charts/carbon-intensity-exporter/values.yaml

```diff
@@ -7,22 +7,35 @@ namespace: kube-system
 
 apiServer:
   name: api-server
-  username: username
-  password: password
   image:
     repository: ghcr.io/azure/kubernetes-carbon-intensity-exporter/server
     pullPolicy: IfNotPresent
     tag: "0.1.0"
+  dataFileLocation: azure-regions.json
+  urls: http://0.0.0.0:7031;http://0.0.0.0:80;
 carbonDataExporter:
   name: carbon-data-exporter
   image:
     repository: ghcr.io/azure/kubernetes-carbon-intensity-exporter/exporter
-    pullPolicy: Always
+    pullPolicy: IfNotPresent
```

Comment:
I needed to build the images locally with arm64 because I'm on mac. This was needed to load the image into kind.

Reply:
This makes sense.